### PR TITLE
fix(blurhash iamge): add missing alt to image

### PR DESCRIPTION
**Pull request by Camc314**
Fix sonarrcloud bug:

 - Add missing `alt` to image

### CODE REVIEW

**Review by ferferga — APPROVED** [review]

File: components/BlurhashImage.vue [hunk 1]
```diff
@@ -15,6 +15,7 @@
         class="absolute"
         :src="image"
         v-bind="$attrs"
+        :alt="item.Name"
```
Comment (MrTimscampi) [inline components/BlurhashImage.vue:18]: I don't think all of them should have a name.

For example, the background image on the home header is only illustrative in purpose. On the other hand, logos should have an alt for screen readers.

**Review by MrTimscampi — CHANGES_REQUESTED** [review]
A better, more flexible way to do this would be to add an `alt` prop, which wouldn't be required and would simply pass its value along to the `alt` property of `img`.

Currently, this implementation is too limiting and doesn't help the users who rely on `alt` to know what they're looking at.

**Comment by Camc314** [comment]
> A better, more flexible way to do this would be to add an `alt` prop, which wouldn't be required and would simply pass its value along to the `alt` property of `img`.

I have updated it so the alt now a prop

**Review by MrTimscampi — APPROVED** [review]
The description aren't ideal (It should describe the content and/or the purpose of the image), but it'll do for now :+1:

**Pull request merged**